custom_containers/pytorch/mnist/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the \"License\");
+# you may not use this file except in compliance with the License.\n",
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an \"AS IS\" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Dockerfile
+FROM python:2.7.15-jessie
```

Comment:
nit: Any reason why we are using Python 2.7 and not 3.5

Reply:
IIRC, `gsutil` only works in Python 2.7. I think that's the only reason I did it.

Reply:
Could do `pip install google-cloud-storage` 

Removes all the gsutil lines in the Dockerfile.

Source code could be similar to:
```
    bucket = storage.Client().bucket(model_dir)
    blob = bucket.blob('{}/{}'.format(
        datetime.datetime.now().strftime('model_name_%Y%m%d_%H%M%S'),
        model_name))
    blob.upload_from_filename(model_name)
```

Reply:
This code may not work locally with 'docker run' if you credential is not setup inside the container. But `gsutil` works in local file system as well, so I can verify the code locally by specifying `--model_dir` to /tmp/<random_model_dir>. If running on the cloud, either way is fine. And if we have other samples using storage client, I think I might keep it as is to show a different option.

Reply:
Sounds good! 
Yea, when running locally, I don't have it upload to cloud storage.